atomix/cluster/src/test/java/io/atomix/raft/RandomizedRaftTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright © 2020 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.atomix.raft;
+
+import com.google.common.io.Files;
+import io.atomix.cluster.MemberId;
+import io.zeebe.util.FileUtil;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import net.jqwik.api.Arbitraries;
+import net.jqwik.api.Arbitrary;
+import net.jqwik.api.EdgeCasesMode;
+import net.jqwik.api.ForAll;
+import net.jqwik.api.Property;
+import net.jqwik.api.Provide;
+import net.jqwik.api.ShrinkingMode;
+import net.jqwik.api.lifecycle.AfterTry;
+import net.jqwik.api.lifecycle.BeforeProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RandomizedRaftTest {
+
+  private static final int OPERATION_SIZE = 10000;
+  private static final Logger LOG = LoggerFactory.getLogger(RandomizedRaftTest.class);
+  public ControllableRaftContexts raftContexts;
+  List<RaftOperation> operations;
+  List<MemberId> raftMembers;
+  File raftDataDirectory;
+
+  @BeforeProperty
+  public void initOperations() {
+    // Need members ids to generate pair operations
+    final var servers =
+        IntStream.range(0, 3)
+            .mapToObj(String::valueOf)
+            .map(MemberId::from)
+            .collect(Collectors.toList());
+    operations = RaftOperation.getDefaultRaftOperations();
+    raftMembers = List.copyOf(servers);
```

Comment:
why we need to copy this here?

Reply:
Not needed. Sorry. left overs from debugging :)